examples/latent_sde.py

```diff
@@ -71,24 +71,47 @@ def f(self, t, y):  # Approximate posterior drift.
         inp = torch.cat((torch.sin(t), torch.cos(t), y), dim=-1)
         return self.net(inp)
 
+    def f_aug(self, t, y):
+        y = y[:, 0:1]
+        f, g, h = self.f(t, y), self.g(t, y), self.h(t, y)
+        z = f - h
+        g = torch.where(
+            g.abs() > 1e-7,
+            g,
+            torch.ones_like(g).fill_(1e-7) * g.sign()
+        )
+        z = z / g
+        z = .5 * (torch.norm(z, dim=1, keepdim=True) ** 2.)
```

Comment:
I don't think the parentheses here are needed, since exponential has higher precedence than multiplication. `z = .5 * torch.norm(z, dim=1, keepdim=True) ** 2` should work.

Reply:
Yeah sure, done!